### PR TITLE
Style changes for the null-view component

### PR DESCRIPTION
**Pull request by kajabijamell**
## Description

Fixes alignment of the null-view component for below the `1200` breakpoint. This will center-align elements and shrink the width of the null-view content container. Also, adds polish for spacing and padding for the null-view component

### Before:
**Desktop View:** 
image
**< Tablet View:**
image

### After
**Desktop View:** 
image
**<Tablet View:** 
image
## Testing in `kajabi-products`
1. Navigate to `podcasts` tab
2. Make sure to delete all podcasts
3. Design should align with screenshots above

LOW, this only changes the visual aesthetic of the null view component and will affects all null view components found in Kajabi Products Repo. In case of more comprehensive regression testing, the implementations of the null view are listed below: 

- `Design` tab, for when an active theme hasn't been selected for new accounts
- `Email Campaigns`
- `Events`
- `Newslwtters`
- `Pages > Landing Pages`
- `Pages > Pipelines`

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼   Great work!

File: packages/sage-assets/lib/stylesheets/components/_null_view.scss [hunk 1]
```diff
@@ -9,25 +9,29 @@
...
+  grid-gap: sage-spacing(lg);
+  justify-content: center;
   max-width: rem(960px);
   margin-left: auto;
   margin-right: auto;
   padding: sage-spacing();
   @media screen and (min-width: sage-breakpoint(lg-max)) {
     grid-template-columns: auto 1fr;
-    grid-gap: 128px;
+    grid-gap: 5rem;
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/components/_null_view.scss:20]: It would be ideal to use either the `sage-spacing()` function here (if a spacer exists for the needed gap) or the `rem()` function to wrap a pixel value (if its different from an existing spacer).

File: packages/sage-assets/lib/stylesheets/components/_null_view.scss [hunk 2]
```diff
@@ -9,25 +9,29 @@
...
 }
 
 .sage-null-view__content {
   display: flex;
-  align-items: flex-start;
+  align-items: center;
   flex-direction: column;
   justify-content: center;
   flex-wrap: wrap;
+  max-width: 450px;
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/components/_null_view.scss:30]: Good place to use `rem()` around this pixel value here.

**Review by philschanely — COMMENTED** [review]
Thanks, @jamell-kajabi ! While the changes make sense and look good I do not see them matching this component in either the [Sage Components (WIP)](https://www.figma.com/file/SCZt9jMd7OcTMrNsOFifDp/?node-id=9%3A156) or the newer [Sage 3 — Admin Figma](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/?node-id=9%3A156) files; these should typically be our source of truth for the built components. Are you working from a source that is expected to be synced into those files or am I perhaps missing something? Thanks in advance! 🙌

File: packages/sage-assets/lib/stylesheets/components/_null_view.scss (hunk 1 above)
Comment (kajabijamell) [reply packages/sage-assets/lib/stylesheets/components/_null_view.scss:20]: Hey Phil, thanks for the review. I don't think we are referring to the same component. I made changes to the `Null View` component but the one referenced is the `Empty State` component. For my changes here, I kept the same horizontal layout for the desktop view and then center-aligned the elements when we get below the `1200` breakpoint. This change was suggested by my team's QA and designer.

**Comment by kajabijamell** [comment]
Hey @philschanely, thanks for the review. I don't think we are referring to the same component. I made changes to the Null View component but the one referenced is the Empty State component. For my changes here, I kept the same horizontal layout for the desktop view and then center-aligned the elements when we get below the 1200 breakpoint. This change was suggested by my team's QA and designer.

**Comment by philschanely** [comment]
@kajabijamell I see your point! The naming is a bit confusing/inconsistent. Here's what is in the 3.0 file for example that I meant to refer to:

Screen Shot 2021-05-05 at 1 37 53 PM

Since we usually refer to these component files for building out or changing things, could your Designer also ensure these changes are made in these shared Figma components?

**Review by philschanely — APPROVED** [review]
Thanks @kajabijamell ! Great clean PR here. Looking forward to syncing up with design on these changes too!
